lib/saddle/version.rb

```diff
@@ -1,3 +1,3 @@
 module Saddle
-  VERSION = '0.0.53'
+  VERSION = '0.1.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.